docs/topics/faq.md

```diff
@@ -87,9 +87,8 @@ In addition, a [command line compiler](command-line.md) is available, which prov
   
 ### What build tools support Kotlin?
 
-On the JVM side, the main build tools include [Gradle](gradle.md), [Maven](maven.md), 
-[Ant](ant.md), and [Kobalt](https://beust.com/kobalt/home/index.html). There are also some build tools available
-that target client-side JavaScript. 
+On the JVM side, the main build tools include [Gradle](gradle.md), [Maven](maven.md), and [Kobalt](https://beust.com/kobalt/home/index.html).
```

Comment:
Last commit to  Kobalt was 3 years ago: https://github.com/cbeust/kobalt

Do we actually want to mention it here? Maybe better to replace it with Amper?

Reply:
Let's remove Ant and Kobalt for now. Amper's status is still not clear, we'll update these recommendations later